doc/environment.yml

```diff
@@ -3,6 +3,7 @@ channels:
   - conda-forge
   - defaults
 dependencies:
+  - python=3.11
```

Comment:
```suggestion
```

Reply:
We do not need this anymore now that we have a release of datatree with the fix.